package.json

```diff
@@ -27,6 +27,7 @@
     "web:dev": "cd web/ && yarn run dev:internal"
   },
   "dependencies": {
+    "@use-gesture/react": "^10.2.20",
```

Comment:
Remove this. This is in the wrong package. Any dependency should be added to web/package.json, not the root package.json

Reply:
Sorry for making this error, now this library is installed and functions properly.